### PR TITLE
[EDE-674] Replace django-test with pytest

### PR DESCRIPTION
**Pull request by tehreem-sadat**
**Description:**

- Replace django-test command with pytest
- Remove "django-nose"
- Add "pytest-cov" and "pytest-django" packages
- Update packages with "make upgrade"

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -3,3 +3,6 @@ current_version = 0.2.0
 commit = True
 
 [bumpversion:file:lang_pref_middleware/__init__.py]
+
+[tool:pytest]
+python_files = test*.py
```
Comment (ziafazal) [inline setup.cfg:8]: why we need `python_files` directive?

File: tox.ini [hunk 2]
```diff
@@ -6,7 +6,9 @@ deps =
     django22: Django>=2.2,<2.3
     django30: Django>=3.0,<3.1
     -r{toxinidir}/requirements/test.txt
-commands = ./manage.py test --with-coverage --cover-inclusive --cover-branches --cover-html --cover-html-dir=build/coverage/html/ --cover-xml --cover-xml-file=build/coverage/coverage.xml --cover-package=lang_pref_middleware
+setenv =
+    DJANGO_SETTINGS_MODULE = settings
+commands = pytest --cov-branch  --cov-report=html:build/coverage/html --cov-report xml:build/coverage/coverage.xml --cov=lang_pref_middleware
```
Comment (ziafazal) [inline tox.ini]: nit: make params consistent. user either `=` or space for all

**Review by ziafazal — CHANGES_REQUESTED** [review]
@tehreem-sadat few minor things

File: setup.cfg (hunk 1 above)
Comment (tehreem-sadat) [reply setup.cfg:8]: Without it, pytest was not collecting tests written in test file (https://github.com/edly-io/django-lang-pref-middleware/blob/master/lang_pref_middleware/tests.py).

File: tox.ini (hunk 2 above)
Comment (tehreem-sadat) [reply tox.ini]: done.

**Review by ziafazal — APPROVED** [review]
👍

**Pull request merged**